P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+            <symbol value="NN" datcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"/>
+            <symbol value="NP" datcat="http://hdl.handle.net/11459/CCR_C-1371_fbebd9ec-a7f4-9a36-d6e9-88ee16b944ae"/>
+            <!-- ... -->
+          </vAlt>
+        </vRange>
+      </fDecl>
+    </egXML>
+    <p>Above, the modeled feature uses <att>targetDatcat</att>, because if it used
+        <att>datcat</att> then we would be saying that the <gi>fDecl</gi> is an instance of the part-of-speech container data category,
+      whereas it is not — the <gi>fDecl</gi> models a container (<gi>f</gi>) that encodes a part
+      of speech. Note also that it is the <gi>f</gi> that is modeled above, not its values, which
+      are used as direct references to data categories, hence the use of <att>datcat</att> in the
+        <gi>symbol</gi> element.</p>
+  </exemplum>
+  
+  <remarks versionDate="2022-09-17" xml:lang="en">
+    <p>The TEI Abstract Model can be expressed as a hierarchy of attribute-value matrices (AVMs)
+      of various types and of various levels of complexity, nested or grouped in various ways. At
+      the most abstract level, an AVM consists of an information container and the value
+      ('contents') of that container.</p>
+    <p>A simple example of an XML serialization of such structures is, on the one hand, the opening
+      and closing tags that delimit and name the container, and, on the other, the content enclosed
+      by the two tags that constitues the value. An analogous example is an
+      attribute name and the value of that attribute.</p>
+    <p>In a TEI XML example of two equivalent serializations expressing the name-value pair
+      <hi style="font-family:monospace">&lt;part-of-speech,common-noun></hi>, namely 
+      <hi style="font-family:monospace">&lt;pos>commonNoun&lt;/pos></hi> and 
+      <hi style="font-family:monospace">pos="common-noun"</hi>, one would
+      classify the element <gi>pos</gi> and the attribute <att>pos</att> as containers (mapping onto
+      the first member of the relevant name-value pair), while the <hi style="font-family:monospace">text()</hi>
+      content of <gi>pos</gi> or the string/symbolic value of <att>pos</att> would be seen as
+      mapping onto the second member of the pair.</p>
+    <p>The <ident type="class">att.datcat</ident> class provides means of addressing the containers
+      and their values, while at the same time providing a way to interpret them in the context of
+      external taxonomies or ontologies. Aligning e.g. both the <gi>pos</gi> element and the
+        <att>pos</att> attribute with the same value of an external reference point (i.e., an entry
+      in an agreed taxonomy) affirms the identity of the concept serialised by both the element container
+      and the attribute container, and optionally provides a definition of that concept (in the case at hand,
+      the concept <soCalled>part of speech</soCalled>).</p>
```

Comment:
_Again_ with the authorial distance! It _is_ the concept “part of speech”, no?

Reply:
Also a term. But, true, no "disclaiming of responsibility" (I should have looked the spec up earlier, rather than going by the wording of so-called soCalled...)